### ISSUE TITLE
Creating the `template.md` file on the fly.

### ISSUE THREAD

**Issue by daffidwilde** [harupy/comment-on-pr#9]
Hi, I'm looking to add an action to a private repository of mine where the word count of the main document is commented when a PR is opened.

I have two components: (1) `wordcount.py` (below) takes the `main.tex` file, summarises the word count and writes it to file at (2) `.github/workflows/wordcount.md`.

```python
# wordcount.py
import os
import subprocess

def main():
    """ Save the output of texcount to file and then trim it. """

    with open(".github/workflows/wordcount.md", "w") as f:
        f.write("The current word count is:\n")
        f.write("==========================\n\n")

        cmd = "texcount -inc -nosub main.tex"
        string = str(subprocess.check_output(cmd, shell=True))
        items = string.split("\\n\\n")
        block = items[-2]

        for line in block.split("\\n")[2:]:
            f.write(line + "\n")


if __name__ == "__main__":
    main()
```

My action file `.github/workflows/wordcount.yml` looks like this then:

```yml
name: Comment word count

on: pull_request

jobs:
  build:

    runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest]
        python-version: [3.7]

    steps:
      - uses: actions/checkout@v1
      - name: Setup Python ${{ matrix.python-version }}
        uses: actions/setup-python@v1
        with:
          python-version: ${{ matrix.python-version }}
      - name: Install LaTeX
        run: |
            sudo apt-get update
            sudo apt-get install -y texlive-extra-utils
      - name: Clone submodules
        run: git submodule update --init --recursive
      - name: Calculate wordcount
        run: python wordcount.py
      - name: Comment word count
        uses: harupy/comment-on-pr@master
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          template: wordcount.md
```

However, when the action runs on GitHub I get the error below and I have no idea why. Would you be able to shed some light on this at all?

```
 with:
    template: wordcount.md
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.7.6/x64
    GITHUB_TOKEN: ************
/usr/bin/docker run --name f64d7289f5ffc4be04e629a4bd1a214c5ca80_982401 \
 --label 1f64d7 --workdir /github/workspace --rm -e pythonLocation -e GITHUB_TOKEN \ 
-e INPUT_TEMPLATE -e INPUT_DOMAIN -e HOME -e GITHUB_REF -e GITHUB_SHA \
-e GITHUB_REPOSITORY -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR \
-e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF \
-e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION \
-e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP \
-e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN \
-e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" \
-v "/home/runner/work/_temp/_github_home":"/github/home" \
-v "/home/runner/work/_temp/_github_workflow":"/github/workflow" \
-v "/home/runner/work/thesis/thesis":"/github/workspace" 1f64d7:289f5ffc4be04e629a4bd1a214c5ca80
python: can't open file 'entrypoint.py': [Errno 2] No such file or directory
```

**Comment by harupy** [comment]
@daffidwilde 
Thanks for reporting this! I just fixed this bug by #10 and it should work now.

**Comment by daffidwilde** [comment]
Brilliant. Thanks so much for your quick response!
